.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - node
+cache:
+  yarn: true
+  directories:
+    - website/node_modules
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+script:
+  - cd website
+  - yarn
+  - yarn lint
+  - yarn watch -- --no-watch
```

Comment:
I guess it's fine for now, I'm working on unit tests, will change it myself if required.